tools/converter/source/onnx/OnnxUtils.cpp

```diff
@@ -20,7 +20,11 @@ bool onnx_read_proto_from_binary(const char* filepath, google::protobuf::Message
     google::protobuf::io::IstreamInputStream input(&fs);
     google::protobuf::io::CodedInputStream codedstr(&input);
 
+#if GOOGLE_PROTOBUF_VERSION >= 3011000
+    codedstr.SetTotalBytesLimit(INT_MAX);
+#else
     codedstr.SetTotalBytesLimit(INT_MAX, INT_MAX / 2);
```

Comment:
The line of SetToalBytesLimit can be deleted.

Reply:
Updated PR